robolectric-processor/src/main/java/org/robolectric/annotation/processing/RobolectricModel.java

```diff
@@ -227,6 +227,11 @@ public void prepare() {
       registerType(solidType);
     }
 
+    for (Map.Entry<TypeElement,TypeElement> entry : getResetterShadowTypes().entrySet()) {
```

Comment:
I'll merge this now as it fixes the breakage I introduced, would you mind adding tests in a follow up PR though?